wfl/DIContainer.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Practices.Unity;
+using Symphono.Wfl.Database;
+
+namespace Symphono.Wfl
+{
+    public class DIContainer
```

Comment:
Smallish nitpick, but this class isn't really a DI container.  It's simply configuring a DI container.  I'm fine with breaking out DI configuration from other application configuration, but we'll want to make sure it takes place within the app startup section of ASP.NET.